contracts/rapidMundaFactory.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+// compiler version must be greater than or equal to 0.8.20 and less than 0.9.0
+pragma solidity ^0.8.20;
+
+
+contract ProcessDefinition {
+    // draft
+    address public factory;
+    constructor() {
+        factory = msg.sender;
+    }
+}
+
+contract RapidMundaFactory {
```

Comment:
Chainflow

Reply:
Should be the ProcessInstance separate contract, or we'll just keep an info about instance in the ProcessDefinition?